test_draft.py

```diff
@@ -143,8 +143,9 @@
     print(events)
 
 
-wumpus_world = create_wumpus_world()
+wumpus_world = create_wumpus_world1()
 wumpus_world = create_wumpus_world2()
+wumpus_world = WumpusWorldGenerator().world
```

Comment:
I would also suggest to add an adhoc test to check the generated grid:

```suggestion
wumpus_world = WumpusWorldGenerator().world
for row in wumpus_world._grid:
    print(row)
```

BTW while the world is generated randomly (check the requested changes in the other comments), it gives an incorrect result, example:

```
> poetry run python test_draft.py
[set(), {<Property.PIT: 3>}, set(), set()]
[set(), {<Property.STENCH: 1>}, set(), {<Property.PIT: 3>}]
[{<Property.PIT: 3>}, {<Property.PIT: 3>}, {<Property.STENCH: 1>, <Property.PIT: 3>, <Property.WUMPUS: 4>}, set()]
[set(), {<Property.PIT: 3>}, {<Property.PIT: 3>}, {<Property.STENCH: 1>, <Property.PIT: 3>}]
```

There is no breeze adjacent to the pits (which makes the world inconsistent). The stench is not adjacent to the wumpus cell.

Reply:
i found the bug, it was this typo in the `_adjacent_coords` function
https://github.com/dpalmasan/wumpus-ai/pull/27/commits/c5367b473ee54d0c59a6f4da9f0438b33aea290d#